Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -406,14 +522,24 @@ def ensure_chrome_running():  # pragma: no cover
             # There's no Chrome listening on that port, Start a new Chrome there
             demisto.debug(f"No Chrome found on port {chrome_port}")
             demisto.debug(f'Initializing a new Chrome session on port {chrome_port}')
-            browser, chrome_port = start_chrome_headless(str(chrome_port))
-            if browser:
-                return browser, chrome_port
+            return chrome_port
 
-        # There's already a Chrome listening on that port, Don't use it
 
-    demisto.error(f'Max retries ({MAX_CHROMES_COUNT}) reached, could not connect to Chrome')
-    return None, None
+def delete_row_with_old_chrome_configurations_from_info_file(info, instance_id, chrome_port):
+    index_to_delete = -1
+
+    info_without_new_lines = info.strip()
+    lines = info_without_new_lines.splitlines()
```

Comment:
You don't need `info_without_new_lines`.
```suggestion
        splitted_info = info.strip().splitlines()
```